apps/web/components/Button.tsx

```diff
@@ -21,50 +21,63 @@ type ButtonProps = {
   walletConnected?: boolean;
 };
 
-type Variant = "primary" | "outline" | "fill" | "error" | "link";
-type VariantStyles = Record<
-  Variant,
-  React.HTMLAttributes<HTMLButtonElement>["className"]
->;
+type Color = "primary" | "secondary" | "tertiary" | "error" | "disabled";
```

Comment:
I think its better naming danger instead of error ?

Reply:
alright i/ll change it to "danger"